### ISSUE TITLE
ItemLevel calculation error

### ISSUE THREAD

**Issue by helicdark** [frozn/TipTac#158]
It seems the 22.12.15 version counted libram's ilvl two times.

**Screenshots**
![WoWScrnShot_121822_114925](https://user-images.githubusercontent.com/12579528/208280534-bd673299-27cc-4203-9561-cf41f400bfd1.jpg)
Libram of Renewal Equipped (ilvl 200), tiptac shows 223 while TinyInspect says 211

![WoWScrnShot_121822_114949](https://user-images.githubusercontent.com/12579528/208280573-38c2d4df-a762-44be-833d-937ae7d6f5dc.jpg)
No Libram, now tiptac is the same as TinyInspect.

**TipTac Reborn (please complete the following information):**
 - Version 22.12.15

**WoW (please complete the following information):**
 - Flavor: WotLKC
 - Version 3.4.0

**Comment by frozn** [comment]
Thanks for reporting this. 👍 

Line 1068 of `\TipTacTalents\libs\LibFroznFunctions-1.0\LibFroznFunctions-1.0.lua` has to be uncommented:

https://github.com/frozn/TipTac/blob/834abf1bdcb31582c6caee5a3289207ed735eef1/TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua#L1065-L1069

The fix will be included in the next release.

**Comment by sharpwg1337** [comment]
> Thanks for reporting this. 👍
> 
> Line 1068 of `\TipTacTalents\libs\LibFroznFunctions-1.0\LibFroznFunctions-1.0.lua` has to be uncommented so that it's active again:
> 
> https://github.com/frozn/TipTac/blob/834abf1bdcb31582c6caee5a3289207ed735eef1/TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua#L1065-L1069
> 
> The fix will be included in the next release.

I uncommented that line and now ilvl doesn't show at all on the tooltip - not blank or N/A, error, etc. it's just gone from the tooltip completely (and yes it is enabled in options)

**Comment by frozn** [comment]
When uncommenting line 1068 you have to add a comma at the end of line 1067. Then it should for work fine.

**Comment by sharpwg1337** [comment]
> When uncommenting line 1068 you have to add a comma at the end of line 1067. Then it should work fine.

OK, now it looks like this:

	local ignoreInventorySlots = {
		[INVSLOT_BODY] = true, -- shirt
		[INVSLOT_TABARD] = true -- tabard,
		[INVSLOT_RANGED] = true -- ranged
	};

comma on line 1067 (1068 is the ranged line) but still not doing anything

**Comment by frozn** [comment]
I meant this (after "true", end of statement before comment):
```
local ignoreInventorySlots = {
	[INVSLOT_BODY] = true, -- shirt
	[INVSLOT_TABARD] = true, -- tabard
	[INVSLOT_RANGED] = true -- ranged
};
```

**Comment by frozn** [comment]
Fixed with release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release).
